### PR TITLE
fix: removed dates from case studies overview page

### PR DESCRIPTION
**Pull request by natapg**
Closes [WEB-323](https://fpjs.myjetbrains.com/youtrack/issue/WEB-323)

### CODE REVIEW

File: src/components/CaseStudy/CaseStudy.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          <h1 className={styles.title}>{title}</h1>
+          <p className={styles.description}>{description}</p>
+        </div>
+      </div>
+    </Link>
+  )
+}
+
+// TODO [VL] Write a proper type for this. It's not as straight forward because it has to accept a lot of different generated gatsby types.
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (tan-90) [inline src/components/CaseStudy/CaseStudy.tsx:28]: Generated Gatsby types still scare me :sadge:

File: src/templates/case-studies.tsx [hunk 2]
```diff
@@ -7,16 +7,18 @@ import { GeneratedPageContext } from '../helpers/types'
...
+import CaseStudy, { mapToCaseStudy } from '../components/CaseStudy/CaseStudy'
 
 interface CaseStudyProps {
   data: GatsbyTypes.CaseStudiesQuery
   pageContext: CaseStudiesContext
 }
 export default function CaseStudies({ data, pageContext }: CaseStudyProps) {
-  const { edges: posts } = data.posts
+  const { edges: caseStudiesData } = data.caseStudies
+  const caseStudies = caseStudiesData.map(({ node }) => node).map((node) => mapToCaseStudy(node))
```
Comment (tan-90) [inline src/templates/case-studies.tsx:21]: Is it possible to do this in a single map? I remember using two maps on the case studies but can't remember if there was a reason.

`const caseStudios = caseStudiesData.map(({node}) => mapToCaseStudy(node))`

**Review by tan-90 — APPROVED** [review]
Just one minor comment, please take a look but I'm approving in advance.

File: src/templates/case-studies.tsx (hunk 2 above)
Comment (natapg) [reply src/templates/case-studies.tsx:21]: yeah, it is possible! I'll change it

**Pull request merged**